services/auto-idler/idle-services.sh

```diff
@@ -49,52 +50,57 @@ echo "$DEVELOPMENT_ENVIRONMENTS" | jq -c '.data.developmentEnvironments[] | sele
         do
           ENVIRONMENT_OPENSHIFT_PROJECTNAME=$(echo "$environment" | jq -r '.openshiftProjectName')
           ENVIRONMENT_NAME=$(echo "$environment" | jq -r '.name')
+          ENVIRONMENT_AUTOIDLE=$(echo "$environment" | jq -r '.autoIdle')
           echo "$OPENSHIFT_URL - $PROJECT_NAME: handling development environment $ENVIRONMENT_NAME"
 
-          # Check if this environment has hits
-          HITS=$(curl -s -u "admin:$LOGSDB_ADMIN_PASSWORD" -XGET "http://logs-db:9200/router-logs-$ENVIRONMENT_OPENSHIFT_PROJECTNAME-*/_search" -H 'Content-Type: application/json' -d'
-          {
-            "size": 0,
-            "query": {
-              "bool": {
-                "filter": {
-                  "range": {
-                    "@timestamp": {
-                      "gte": "now-4h"
+          if [ "$ENVIRONMENT_AUTOIDLE" == "0" ]; then
+            echo "$OPENSHIFT_URL - $PROJECT_NAME: $ENVIRONMENT_NAME idling disabled, skipping"
+            continue
```

Comment:
let's keep the `if` crazyness here down, it's already wrapped in many ifs in here. 

If we use `continue` we don't need an `else`, as the `continue` anyway will stop executing and go to the next loop.

So we can just run and `fi` instead of an `else`